src/instruments/src/PFD/shared/ArincValueProvider.ts

```diff
@@ -148,4 +188,17 @@ export class ArincValueProvider {
 
         publisher.pub('chosenRa', chosenRadioAltitude);
     }
+
+    private determineAndPublishChosenLandingElevation(publisher: Publisher<Arinc429Values>) {
+        const useOpposite = this.ownLandingElevation.isFailureWarning()
+            && this.ownLandingElevation.isNoComputedData()
+            && !this.oppLandingElevation.isFailureWarning()
+            && !this.oppLandingElevation.isNoComputedData();
```

Comment:
Looks like this is always false? Probably meant
```suggestion
        const useOpposite = (this.ownLandingElevation.isFailureWarning()
            || this.ownLandingElevation.isNoComputedData())
            && !this.oppLandingElevation.isFailureWarning()
            && !this.oppLandingElevation.isNoComputedData();
```

Reply:
Oof, brilliant catch. 👍